moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/bullet_integration/basic_types.h

```diff
@@ -33,6 +33,9 @@
 
 namespace collision_detection_bullet
 {
+
+static const rclcpp::Logger BULLET_LOGGER = rclcpp::get_logger("collision_detection.bullet");
```

Comment:
This has been addressed.  I couldn't use extern as I had to declare this global variable in each cpp file that needed it and so having extern would cause a multiple definition build error.